docs/howtos/layers/index.md

```diff
@@ -7,13 +7,5 @@ following how-to guides:
 ```{tableofcontents}
```

Comment:
@melissawm do you want to update this one also?

Reply:
I was actually going to keep this one as I think it's useful in this page. Do you have another suggestion for linking to these sub-pages?